rainier-core/src/main/scala/com/stripe/rainier/core/Model.scala

```diff
@@ -14,7 +14,8 @@ case class Model(private[rainier] val likelihoods: List[Real],
       implicit rng: RNG): Trace = {
     val range =
       if (parallel)
-        1.to(nChains).par
+        // TODO this used to return a ParRange. To do that in 2.13 we would need scala-parallel-collections
+        1.to(nChains)
```

Comment:
Decide if we still need `parallel` support and then either clean up or bring in parallel collections.

Reply:
We can drop it for now.